modules/ROOT/pages/shared-bucket-access.adoc

```diff
@@ -6,7 +6,7 @@ _(Introduced in Sync Gateway 1.5 and Couchbase Server 5.0)_
 Shared Bucket Access enables document changes made through Couchbase SDKs and N1QL queries to be synchronized to Couchbase Lite clients, and vice versa.
 The following diagram shows different clients (mobile, web, desktop) writing data to the same bucket in Couchbase Server.
 
-image::shared-bucket-access.svg[]
+image::shared-bucket-access.png[]
```

Comment:
1. Is there a new text source file for this image so others can edit it in future?
2. See my question about port 80 on #113

Reply:
1. Yes, I've checked it in with this PR in the `images` folder. https://github.com/couchbase/docs-sync-gateway/pull/121/commits/9e0683ce0ccc69b7c2d8ca6321cdc4f9d47a989d?short_path=9a4d46a#diff-9a4d46adb644c664c142d9aee3550367
2. Same as on PR 113. Updated diagram with the "port" prefix.

Thanks!